src/libraries/Common/tests/System/Net/EnterpriseTests/setup/apacheweb/run.sh

```diff
@@ -1,5 +1,21 @@
-#!/bin/sh
+#!/bin/bash -x
+
+echo "$@" > /tmp/args
 
 cp /SHARED/apacheweb.keytab /etc/krb5.keytab
 
-exec httpd -DFOREGROUND "$@"
+if [ "$1" == "-debug" ]; then
+  while [ 1 ];do
+    sleep 10000
+  done
+fi
+
+if [ "$1" == "-DNTLM" ]; then
+  ./setup-pdc.sh
+  /usr/sbin/apache2 -DALTPORT "$@"
+  shift
```

Comment:
Do I get it right that the goal here is to start two apache instances? (This one in the background.)

Can you add an explanation why are we doing so?

Reply:
Yes. we did run two instances before but the mechanic was different. This comes from the fact that we use single docker image to run standard and non-default service.